apps/temp-docs/components/Navigation/SideBar.tsx

```diff
@@ -2,21 +2,16 @@ import Link from 'next/link'
 import Image from 'next/image'
 import { useRouter } from 'next/router'
 import { IconChevronRight, IconArrowLeft } from '~/../../packages/ui'
```

Comment:
can do this in another commit, but what's that thing we need to do to make it so it's just `import {thing} from 'ui'` ?

Reply:
oh yeah i'll do a general clean up of these - i think its already set up to do so